tests/stringnumber_tests.ts

```diff
@@ -275,8 +275,10 @@ function multi(options: TestOptions): void {
 
 function drawAccidentals(options: TestOptions): void {
   const f = VexFlowTests.makeFactory(options, 750);
-  const glyphScale = 39; // default font scale
-  const clefWidth = Glyph.getWidth('gClef', glyphScale); // widest clef
+  const el = new Element();
+  el.setText(String.fromCharCode(0xe050));
```

Comment:
Can we just use the unicode string directly instead of converting from char code? Also, we should include the SMuFL glyph name for future reference. Same comment goes for the above, in ornament_tests.ts.